### DISCUSSION TITLE
Email verification on user registration

### DISCUSSION THREAD

**Discussion by hxuanhung** [fief-dev/fief#174] [Q&A]
I would like to ask user to verify their email using a code sent to their email once they signed up. It seems not possible with the current version, isn't it?

**Comment by frankie567** [comment]
Currently, we do not have a mechanism to very email address, but this is definitely on the short-term roadmap!

**Comment by martincpt** [comment]
It would be neat to have a config for this, either to send the verification email right after registration or send it at the first login attempt.
